.github/workflows/linux.yml

```diff
@@ -380,7 +380,8 @@ jobs:
 
   ONNX_Runtime:
     name: ONNX Runtime Integration
```

Comment:
Jus for completeness, `linux_arm64.yml` also requires the same change

Reply:
Added in PR https://github.com/openvinotoolkit/openvino/pull/21931